src/url.js

```diff
@@ -92,12 +92,15 @@ export function addParamsToUrl(url, params) {
  * Provides an exception for localhost.
  * @param {string} urlString
  * @param {!Element} elementContext Element where the url was found.
+ * @param {!Window=} opt_win
  * @return {string}
  */
-export function assertHttpsUrl(urlString, elementContext) {
+export function assertHttpsUrl(urlString, elementContext, opt_win) {
```

Comment:
Will we actually ever supply `opt_win`? It seems a bit odd, given that all URLs are resolved against the current window in the line above `parseUrl(urlString)`. Or is it only for tests?